data/base-spells.yaml

```diff
@@ -149,81 +149,81 @@ khri_preps:
 
 barb_abilities:
   Python:
-    activate: form python
-    start: imitating the cunning ways of the python
-    expire: finish practicing the Form of the Python
+    start_cmd: form python
```

Comment:
Used this chance to make them past tense. It's really only a matter of copy and replacing in my open files, so no big deal! :)